python/lsst/ts/m2gui/actuator_force.py

```diff
@@ -0,0 +1,68 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ActuatorForce"]
+
+from dataclasses import dataclass, field
+
+from . import NUM_ACTUATOR
+
+
+@dataclass
+class ActuatorForce:
+    """Actuator force class to have the force details contain the look-up table
+    (LUT) information."""
+
+    # The unit of force is Newton.
+
+    # Force component in the calculation of gravity LUT.
+    f_e: list = field(default_factory=lambda: [0] * NUM_ACTUATOR)
```

Comment:
Good, but even better would be:
```python
from typing import List
.....
    f_e : List[int] = field(default_factory=lambda: [0] * NUM_ACTUATOR)
```

Reply:
Improved in the commit: [7127817](https://github.com/lsst-ts/ts_m2gui/pull/8/commits/7127817f46a8471c10748ab039065b7f9df0c9b1). Thanks!